core/templates/pages/exploration-player-page/services/player-correctness-feedback-enabled.service.ts

```diff
@@ -24,7 +24,7 @@ import { downgradeInjectable } from '@angular/upgrade/static';
   providedIn: 'root'
 })
 export class PlayerCorrectnessFeedbackEnabledService {
-  static _correctnessFeedbackIsEnabled: boolean = false;
+  static _correctnessFeedbackIsEnabled: boolean = true;
```

Comment:
Why do we still need this variable (and this entire service)?

Reply:
The service is obsolete now, so I dropped it